### DISCUSSION TITLE
Is 5.5 available through vcpkg?

### DISCUSSION THREAD

**Discussion by NoudL** [CGAL/cgal#6820] [Q&A] [answered]
Hi,

I'd really like to update CGAL so that I could use the Edges range type for Polygon_2, I've installed CGAL through vcpkg and I believe 5.5 should be available, but ```vcpkg upgrade``` only lists up to 5.4.1 . Am I doing something wrong or is vcpkg behind?

The version listed on [microsoft/vcpkg/ports/cgal/CONTROL](https://github.com/microsoft/vcpkg/blob/030cfaa24de9ea1bbf0a4d9c615ce7312ba77af1/ports/cgal/CONTROL) is even older (5.2).

But seeing [this commit ](https://github.com/microsoft/vcpkg/commit/6def72be5a2807be3b5f838f713ee792edf264c8) it should be updated there, so I feel like I'm missing something/doing something wrong.

Any help is appreciated!

**Comment by lrineau** [comment, marked as answer]
CGAL 5.5 is indeed updated to version 5.5, in vcpkg. And the `CONTROL` file no longer exists. The latest version of the port is https://github.com/microsoft/vcpkg/tree/master/ports/cgal.

You need to update your local copy of vcpkg: see [How do I update vcpkg?](https://github.com/microsoft/vcpkg/blob/master/docs/about/faq.md#how-do-i-update-vcpkg).

**Reply by NoudL** [reply]
thank you very much, I knew I was overlooking something simple
